azure/notebooks/kubernetes/Detailed_setup_guide.md

```diff
@@ -0,0 +1,213 @@
+# [Detailed Guide to use Dask on Azure Kubernetes Service (AKS)](#anchor-start)
+
+For all the next steps, we will be using the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli), however the same can be achieved through the Azure GUI Portal. 
+
+### [Step 0: Install and authenticate with Azure CLI](#anchor-install-azurecli)
+- Install the `az` cli using 
+    ```
+    curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+    ````
+    in the computer from where you will be running everythning from. Get rid of the `sudo` if you are running things from inside a container. 
+
+- Once `az` is installed, make sure you you configure the local `az` cli to work with your Azure credentials, do `az login` and authenticate from Microsoft's website.
+
+- For more details follow the steps [Here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). 
+
+
+### [Step 1: Install Kubectl](#anchor-install-kubectl)
+
+- Install `kubectl` to access your cluster from the local machine from the following link : [https://kubernetes.io/docs/tasks/tools/] depending on your operating system. 
+
+
+### [Step 2: Set some environment variables](#anchor-set-env-variables)
+We will set some environment variables beforehand, namely to help us deploy some resources quickly.  We will keep on using some common values for the rest of the deployments. . 
+```bash
+REGION_NAME=<your preferred location>
+RESOURCE_GROUP=<your preferred resource group name>
+SUBNET_NAME=<optional subnet name>
+VNET_NAME=<optional vnet name>
+VERSION=$(az aks get-versions --location $REGION_NAME --query \
+                'orchestrators[?!isPreview] | [-1].orchestratorVersion' \--output tsv)
+AKS_CLUSTER_NAME=<your cluster name>
+VM_SIZE=Standard_NC12s_v3 # or any other VM size. We use VMs with GPU
+```
+
+- We would need a resource group. You can create a resource group in Azure `az group create  --name $RESOURCE_GROUP --location $REGION_NAME`, or use an existing one.
+
+- Secondly, we get the latest non preview Kubernetes version and store it in an env variable using. This provides that the latest version is 1.20.5. Optionally you can directly set the Kubernetes version in the environment variable `VERSION`.
```

Comment:
Were you planning to add anything after using?

Reply:
This is a good catch. Left over from restructuring. Fixed.